gym/wrappers/test_normalize_action.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+import gym
+from gym.wrappers import NormalizeAction
+
+
+def test_normalize_action():
+    env = gym.make('CartPole-v1')
+    with pytest.raises(AssertionError):
+        env = NormalizeAction(env)
+    del env
+
+    env = gym.make('Pendulum-v0')
+    env = NormalizeAction(env)
+    env.reset()
+    with pytest.raises(AssertionError):
+        env.step(10+env.action_space.sample())
```

Comment:
nitpicking, but I would prefer a little more testing - say, for pendulum action limits are [-2, 2], so make sure action 1.5 causes an error after wrapping; and before wrapping produces the same observation as action 0.75 after wrapping?